integration-tests/tests/app.suite.ts

```diff
@@ -1,10 +1,14 @@
 import { expect } from 'chai'
 import { Server } from 'hapi'
 import { Bootstrap } from '../shared/bootstrap'
+import { Requester } from '../shared/requester'
+
+var requester: Requester
 
 describe('App', () => {
 
   before(() => {
+    requester = new Requester('http://127.0.0.1:3000')
```

Comment:
don't want to create a config file just for that url for now. Probably doing it next time :)